src/test/regress/expected/multi_complex_expressions.out

```diff
@@ -473,7 +473,6 @@ SELECT o_custkey FROM orders GROUP BY o_custkey LIMIT 10 OFFSET 15;
 -- GROUP BY and ORDER BY non-aggregate -> push-down
 CREATE TEMP TABLE temp_limit_test_3 AS
 SELECT o_custkey FROM orders GROUP BY o_custkey ORDER BY o_custkey LIMIT 10 OFFSET 15;
-DEBUG:  push down of limit count: 25
```

Comment:
I think this is because we're no longer pushing down grouping since we have to group in master query regardless